internal/server/data/data.go

```diff
@@ -390,3 +405,35 @@ func InfraConnectorIdentity(db ReadTxn) *models.Identity {
 	}
 	return connector
 }
+
+const slowQueryThreshold = time.Second
+
+func logQuery(query string, err error, startedAt time.Time, rows int64) {
+	level := zerolog.TraceLevel
+
+	elapsed := time.Since(startedAt)
+	switch {
+	case errors.Is(err, sql.ErrNoRows), errors.Is(err, gorm.ErrRecordNotFound):
+		level = zerolog.WarnLevel
```

Comment:
Previously we were using `ErrorLevel` for queries that errored with "No Rows". I've changed it to WARN here, since a not found doesn't seem like a critical error.